index.html

```diff
@@ -1,289 +1,307 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
 
 
     <link rel="stylesheet" href="index.css">
-    <title>skeleton mobile</title>
+    <title>Roger Software developer </title>
 </head>
+
```

Comment:
[OPTIONAL] Kindly change the contrast to be at least `4.5` for these failed elements because they failed in the contrast check
![screencapture-aditus-io-button-contrast-checker-rawcdn-githack-com-2022-03-30-at-01-00-39-575-2022-03-30-03_07_31](https://user-images.githubusercontent.com/24703171/160731019-d57545c0-86c9-411f-901e-d130d9b676b1.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)

Reply:
Thank you very much for the link